WFInfo/CustomEntrypoint.cs

```diff
@@ -240,7 +271,7 @@ private static void DownloadProgressCallback(object sender, DownloadProgressChan
             dialogue.Dispatcher.Invoke(() => { dialogue.UpdatePercentage(e.ProgressPercentage); });
         }
 
-        private static async void RefreshTesseractDlls(CancellationToken token)
+        private static async void RefreshTesseractDlls(CancellationToken token, bool AvxSupport)
```

Comment:
outside perspective, but `AvxSupport` doesn't seem to be used or passed down in here

Reply:
It wasn't used in very straightforward way (felt like global scope), so I decided to make it more clear - its local variable at least.
It shouldn't functionally affect anything.

Reply:
Its used [here](https://github.com/WFCD/WFinfo/blob/c3441f114748e4bc112e16940cae1cffc812936e/WFInfo/CustomEntrypoint.cs#L303)